test_autoflake.py

```diff
@@ -1224,6 +1224,43 @@ def test_exclude(self):
         finally:
             shutil.rmtree(temp_directory)
 
+    def test_with_ignore_init_module_imports_flag(self):
```

Comment:
this section of the test module contains test methods about file operations.  your test modules should be either up to top near the other import methods, or near another similar type of test methods.

Reply:
Moved!